core/prisma/scripts/history-tables/history-table-migration-template.sql

```diff
@@ -0,0 +1,67 @@
+/*
+ Warnings:
+
+ - Added the required column `lastModifiedBy` to the `{{tableName}}` table without a default value. This is not possible if the table is not empty.
+ */
+-- AlterTable
+-- first we add the column with a default value
+ALTER TABLE "{{tableName}}"
+  ADD COLUMN "lastModifiedBy" modified_by_type NOT NULL DEFAULT CONCAT('unknown', '|', FLOOR(EXTRACT(EPOCH FROM CURRENT_TIMESTAMP) * 1000));
+
+-- then we remove the default value
+ALTER TABLE "{{tableName}}"
+  ALTER COLUMN "lastModifiedBy" DROP DEFAULT;
```

Comment:
we first set `lastModifiedBy` to `unknown|${Date.now()}` for all rows
then we drop the default